### ISSUE TITLE
Open in Google Drive not working

### ISSUE THREAD

**Issue by cangevine** [sprang/Inkpad#134]
I am unable to open in Google Drive. The blank preview window appears as described in [issue #128](https://github.com/sprang/Inkpad/issues/128), which is not a problem, then I use the Share icon and "Open in Google Drive." The Google Drive app switches in -- so far everything seems good -- but then the .svg never appears anywhere. I don't see it in the Google Drive app that just switched in, and I can't find it from a desktop looking for recent files in my Drive.

Any advice?

**Comment by alistairmcmillan** [comment]
I'm able to replicate the issue exactly as you describe. When I export to Dropbox it accepts the file and asks me where I want to save it within my Dropbox, but when I try to export to Google Drive it just opens the Google Drive app. I've tried exporting to Google Drive from a few apps and nothing seems to happen so this looks like a Google Drive problem, rather than something Inkpad is doing wrong. Could you try exporting to Google Drive from another app and see if you get the same fault?

BTW iOS didn't support SVG natively before, but as of iOS 8 it does seem to support SVG because now I get SVG previews when I try to export from Inkpad. :)

**Comment by cangevine** [comment]
I just tested opening a PDF on my iPhone from Mailbox into Google Drive.
Despite all of the variables I'm changing here, it worked fine and prompted
me to upload, [just like this tutorial tells you to expect](https://snapguide.com/guides/save-a-pdf-to-your-google-drive-from-your-ipad/).
I suppose this is in the intended "Open in Drive" behavior.

On Sat, Oct 4, 2014 at 12:58 PM, Alistair McMillan <notifications@github.com

**Comment by alistairmcmillan** [comment]
Yes. I tried send a PDF on my iPhone from Mailbox to Google Drive and it worked okay too. However I can't get a PDF on my iPad from Mailbox to Google Drive. Can you?

**Comment by cangevine** [comment]
Update: I have been using Inkpad with a number of my students, and today I
had a few of them experiment with Opening in Google Drive... and it worked!
It opened the app and prompted them to upload. I wasn't able to sort out
any patterns about OS version... but will try to take another look
tomorrow.
